api/all/src/main/java/io/opentelemetry/api/OpenTelemetry.java

```diff
@@ -39,6 +40,9 @@ static OpenTelemetry propagating(ContextPropagators propagators) {
   /** Returns the {@link TracerProvider} for this {@link OpenTelemetry}. */
   TracerProvider getTracerProvider();
 
+  /** Returns the {@link MeterProvider} for this {@link OpenTelemetry}. */
+  MeterProvider getMeterProvider();
```

Comment:
Isn't it a breaking change to add this without a `default` method?

Reply:
Oops thanks! Hmm thought the japi check would have caught this too bad